### ISSUE TITLE
Better wiki information

### ISSUE THREAD

**Issue by mebsim** [nu-dev/nu#18]
Wiki should be worked on.

**Comment by ohnx** [comment]
Indeed, will work on once I find the time (likely once school starts)

**Comment by mebsim** [comment]
Thanks for a bit of an update.

**Comment by mebsim** [comment]
Still might want a bit more detail.

**Comment by ohnx** [comment]
ya, ya, ya

wait until school starts and i start finding the time to do stuff :stuck_out_tongue_closed_eyes:

**Comment by mebsim** [comment]
That makes sense. Wait until you have your days taken up by school to finish your work. I completely understand.
